app/src/main/java/com/example/football_manager/News/CreateNews.kt

```diff
@@ -0,0 +1,94 @@
+package com.example.football_manager.News
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+
+
+@Composable
+fun CreateNews(navController: NavHostController) {
+    val errors = remember { mutableStateListOf<String>() }
+    val NEWS_TITLE_MIN_LENGTH = 3
+    val NEWS_TITLE_MAX_LENGTH = 30
+    val NEWS_WRITER_NAME_MIN_LENGTH = 4
+    val NEWS_WRITER_NAME_MAX_LENGTH = 10
+    val NEWS_CONTENT_MIN_LENGTH = 10
+    val NEWS_CONTENT_MAX_LENGTH = 120
```

Comment:
120 characters is too short

Reply:
300?

Reply:
Tänk att man ska kunna skriva typ en artikel, truppen till match etc. 
Nästan man man inte behöver sätta någon max längd.

Reply:
Ok!